src/context/cartContext.js

```diff
@@ -0,0 +1,27 @@
+import { createContext, useContext, useReducer } from "react";
+import { initialSharedState } from "../reducers/constants";
+import { sharedReducer } from "../reducers/reducerFunctions";
+
+/*
+ *   Intial Shared State
```

Comment:
Don't leave commented code as tanay said😁 it looks dirty. So please take care of that.

Reply:
Actually Its for Information as I have used the same shared state so It will be easy to know what state contains.